uptime/monitor-setup/checks.json

```diff
@@ -469,31 +465,86 @@
       "msp_num_retries": 2,
       "msp_use_ip_version": "",
       "msp_uptime_sla": "0.9900",
+      "msp_response_time_sla": "6.000",
+      "msp_notes": "",
+      "msp_include_in_global_metrics": true,
+      "stats_url": "https://uptime.com/api/v1/checks/1388584/stats/",
+      "analysis_url": "https://uptime.com/api/v1/checks/1388584/analysis/",
+      "alerts_url": "https://uptime.com/api/v1/alerts/?check_pk=1388584"
+    },
+    {
+      "pk": 1567750,
+      "url": "https://uptime.com/api/v1/checks/1567750/",
+      "name": "PUT - Artifactory CLAB",
+      "cached_response_time": 0.272834333333333,
+      "contact_groups": [
+        "platform-services-dxc"
+      ],
+      "created_at": "2023-04-04T22:17:24.533174Z",
+      "modified_at": "2023-04-25T18:57:02.895373Z",
+      "locations": [
+        "US West",
+        "US Central",
+        "Canada"
+      ],
+      "tags": [
+        "shared-services"
+      ],
+      "check_type": "HTTP",
+      "escalations": [],
+      "maintenance": {
+        "state": "ACTIVE",
+        "schedule": []
+      },
+      "monitoring_service_type": "HTTP",
+      "is_paused": false,
+      "is_under_maintenance": false,
+      "state_is_up": true,
+      "state_changed_at": "2023-04-22T17:19:37.121392Z",
+      "msp_protocol": "",
+      "msp_interval": 10,
+      "msp_address": "https://n8n-bf5ef6-dev.apps.gold.devops.gov.bc.ca:443/api/v1/executions?workflowId=3&limit=1",
+      "msp_port": 443,
+      "msp_username": "",
+      "msp_proxy": "",
+      "msp_dns_server": "",
+      "msp_dns_record_type": "",
+      "msp_status_code": "",
+      "msp_send_string": "",
+      "msp_expect_string": "\"finished\":true",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "SSL_TLS",
+      "msp_threshold": 40,
+      "msp_headers": "X-N8N-API-KEY: n8n_api_1a70d4baeef73d67f0b98e2e8a40095f716b044db7ea701dd5862d1c2727331cf8a4312b2576f999",
```

Comment:
this is a public repo, please remove sensitive info from the payload! you can keep them in the team shared Vault space!